app/controllers/users/sessions_controller.rb

```diff
@@ -28,6 +28,7 @@ def new
       @request_id = request_id_if_valid
       @ial = sp_session_ial
       @browser_is_ie11 = browser_is_ie11?
+      @a_b_test_bucket = AbTests::SIGN_IN.bucket(session.id)
```

Comment:
can we rename this ivar too? (and in `sessions/new.html.erb` too)
```suggestion
      @sign_in_a_b_test_bucket = AbTests::SIGN_IN.bucket(session.id)
```

Reply:
Updated in c0649e70f89e9bffa6942206ac05ecdf18447ff6